2022q4/pytest-for-atf

```diff
@@ -0,0 +1,21 @@
+=== Pytest support for the FreeBSD testing framework
+
+Links: +
+link:https://reviews.freebsd.org/D31084[Initial review] URL: link:https://reviews.freebsd.org/D31084[https://reviews.freebsd.org/D31084] +
+link:https://cgit.freebsd.org/src/tree/tests/examples/test_examples.py[Test examples] URL: link:https://cgit.freebsd.org/src/tree/tests/examples/test_examples.py[https://cgit.freebsd.org/src/tree/tests/examples/test_examples.py]
+
+Contact: Alexander Chernikov <melifaro@FreeBSD.org>
+
+Native pytest support for atf(7) enhances the capabilities of the FreeBSD testing.
+
+Pytest simplifies the test writing by reducing the amount of boiler-plate code.
+It offers several advantages over the existing atf-c and atf-shell bindings.
+One of the most important ones is the test parametrization, which allows to improve coverage without writing nearly any code. The other is rich assert system, offering detailed errors description.
```

Comment:
s/the test/test/

s/allows to improve/allows improving/

s/without writing nearly any code/with writing nearly no code/

s/is rich/is a rich/